.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-name: Tests
+name: ci
```

Comment:
Now CI has more than just tests.

Also this shorter prefix makes it a little easier to read in the UI:

<img width="869" alt="image" src="https://github.com/user-attachments/assets/3e087fa3-b1e6-4977-8a14-6be0d7b17438" />